libs/workspace-manager/src/lib/ui/filesystem-navigator/filesystem-navigator.component.scss

```diff
@@ -0,0 +1,20 @@
+$list-item-hover: rgba(222, 222, 222, 0.7);
+
+:host {
+  .list-wrapper {
+    overflow: auto;
+    margin-top: 16px;
+    height: inherit;
+    max-height: 100%;
+  }
+
+  mat-list-item:hover {
+    background: $list-item-hover;
+    cursor: pointer;
+  }
+
+  .angular-logo {
+    width: 25px;
+    height: 25px;
+  }
+}
```

Comment:
@Danieliverant Is this the way to use material colors in this scenario?

Reply:
no... why do you need to change the color? @tomzo42 
every material override should be done on the theme level not single component

Reply:
@Danieliverant I wanted to add a hover state for the list item, it doesn't have one by default

Reply:
@Tcodrz I don't think it's needed 
see material example:
https://material.angular.io/components/list/examples#list-sections

if you really want it you can create list.scss file in the theme, and override it there, if you want ping me on Slack we can do it together

Reply:
doesn't seem to have that hover effect like in the material example. 
IMO the list looks not so interactive without the hover, if I will use the theme for getting a grey color that would be OK or better to just leave it for now ?
@Danieliverant

Reply:
I guess it cuz you need to put `matListItemIcon` on the `mat-icon` itself and not on it's parent.

It's better to fix this ^ and leave it without a dedicated hover for now.
But if you want, you can override the hover behavior in the theme files

Reply:
@Danieliverant doesn't work either, I removed the css for now, will check it out in a different PR